### ISSUE TITLE
assetfinder in osx 10 

### ISSUE THREAD

**Issue by lordpython757** [tomnomnom/assetfinder#19]
Hello guys im a total noob most experience i have is learning python, anyways when trying to run 
assetfinder [--subs-only] <domain> it says 
users-MacBook-Pro:assetfinder-master user$ assetfinder -subs-only twitter.com
-bash: assetfinder: command not found

**Comment by cybercdh** [comment]
have you installed GO and have you added GO to your path environment variable?

**Comment by lordpython757** [comment]
yes i have go.  export PATH=$PATH:/assetfinder is that the command i ran it and got this sers-MacBook-Pro:~ user$ echo $PATH
/Library/Frameworks/Python.framework/Versions/3.7/bin:/Library/Frameworks/Python.framework/Versions/3.8/bin:/usr/local/bin:/usr/local/bin:/Library/Frameworks/Python.framework/Versions/3.5/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/X11/bin:/usr/local/go/bin:/usr/local/assetfinder/bin:/usr/bin/assetfinder. but it still said asserfinder is not a command

**Comment by cybercdh** [comment]
you don't need to include assetfinder in you PATH, it's go itself.

export PATH=$PATH:/usr/local/go/bin

Perhaps also take a look at the docs:

https://golang.org/doc/install

**Comment by cybercdh** [comment]
Also, did you run 

`go get -u github.com/tomnomnom/assetfinder`

to install assetfinder, or did you clone the repo locally?

**Comment by dipanshu-mewara** [comment]
Hey ,Does cloning make any difference?After cloning how to use assetfinder?

**Comment by dipanshu-mewara** [comment]
Its showing Cannot download when I ran above cmd

**Comment by D33van** [comment]
will this tool work for mac ??

**Comment by tobiaskohler** [comment]
> will this tool work for mac ??

Hey D33van, sure, it works on MacOS. Just follow tomnomnoms instructions, install it via "go get ..." and do not forget to put it into $PATH to make execution a bit more easier.

**Comment by devcoinfet** [comment]
> `export PATH="$PATH:/home/USERNAME/go/bin"`

your a damn genius thank u

**Comment by guchorov** [comment]
Can you give the command for MacOS. in detail

**Comment by bhavyayaya** [comment]
go install github.com/tomnomnom/assetfinder@latest
